src/project-css/.env.circleci

```diff
@@ -31,5 +31,5 @@ PUSHER_APP_ID=
 PUSHER_KEY=
 PUSHER_SECRET=
 
-TIKA_HOST=tika_service
-TIKA_PORT=9998
+TIKA_HOST=127.0.0.1
```

Comment:
Why change that to a local IP instead of using the docker syntax?  The one concern I have is that you would be using the docker containers loopback IP instead of the home systems loopback IP.  That is why in some setups I've had to change this to 0.0.0.0.  I think your better off using the docker container name though.

Reply:
Circleci wasn't able to connect to the container using a host name so I used 127.0.0.1 and it worked. I can try 0.0.0.0